backend/WebApi/Controllers/TestingController.cs

```diff
@@ -11,23 +13,34 @@ namespace LanguageForge.WebApi.Controllers;
 public class TestingController : Controller
 {
     private readonly JwtService _jwtService;
+    private readonly UserService _userService;
 
-    public TestingController(JwtService jwtService)
+    public TestingController(JwtService jwtService, UserService userService)
     {
         _jwtService = jwtService;
+        _userService = userService;
     }
 
-    [HttpGet("make-jwt")]
+    [HttpGet("make-jwt-role/{role}")]
     [AllowAnonymous]
-    public string MakeJwt(UserRole role)
+    public string MakeJwtForTole(UserRole role)
     {
         return _jwtService.GenerateJwt(
             new LfUser("test@test.com",
                 LfId.FromDb<User>(ObjectId.GenerateNewId()),
-                role)
+                role,
+                new[] { new UserProjectRole("test-3", ProjectRole.Manager) })
         );
     }
 
+    [HttpGet("make-jwt-email/{email}")]
+    [AllowAnonymous]
+    public async Task<ActionResult<string>> MakeJwtForUser([EmailAddress] string email)
```

Comment:
oh nice attribute, does it actually throw if you don't pass in an email?

Reply:
Indeed it does 🤓 
```
{
  "type": "https://tools.ietf.org/html/rfc7231#section-6.5.1",
  "title": "One or more validation errors occurred.",
  "status": 400,
  "traceId": "00-a320d2c563dc5376c7c82d67db17dd9b-65984fc47359c38d-00",
  "errors": {
    "email": [
      "The email field is not a valid e-mail address."
    ]
  }
}
```